evergreen.yaml

```diff
@@ -26,26 +26,18 @@ variables:
 #######################################
 functions:
   run-make:
-    command: shell.exec
+    command: subprocess.exec
     type: test
     params:
-       script: |
-         set -o errexit
-         set -o verbose
-
-         # Configure a local go path for this build.
-         export GOPATH=`pwd`/gopath
-
-         # on windows we need to turn the slashes the other way
-         if [ "Windows_NT" == "$OS" ]; then
-            export GOPATH=$(cygpath -m $GOPATH)
-         fi
-
-         cd gopath/src/github.com/mongodb/jasper
-
-         # Run make, called with proper environment variables set,
-         # running the target.
-         ${build_env|} make -k ${target}
+      working_dir: gopath/src/github.com/mongodb/jasper
+      binary: make
+      args: ["${make_args|}", "${target}"]
+      env:
```

Comment:
if you set the expansions correctly, you can use the "expose_expansions_to_env" option, to reduce the boilerplate